src/main/java/org/synyx/urlaubsverwaltung/application/application/ApplicationForLeaveViewController.java

```diff
@@ -131,7 +131,7 @@ private static ApplicationForLeaveDto toView(ApplicationForLeave application, Pe
             .workDays(decimalToString(application.getWorkDays(), locale))
             .statusWaiting(isWaiting)
             .editAllowed(isWaiting && person.equals(signedInUser))
-            .approveAllowed(canAllow && (isBoss || !person.equals(signedInUser)))
+            .approveAllowed(canAllow && !person.equals(signedInUser))
             .temporaryApproveAllowed(canAllow && (isBoss || !person.equals(signedInUser)) && isDepartmentHead && twoStageApproval && isWaiting)
             .rejectAllowed(canAllow && (isBoss || !person.equals(signedInUser)))
```

Comment:
Ist das auch bei der vorläufigen Freigabe und dem Ablehnen so, dass man das selbst nicht machen darf? 

Das Template scheint an der Stelle das Problem aktuell abzufangen: https://github.com/synyx/urlaubsverwaltung/blob/9b6641de5a474a8f1800029ba337f0d448feacf4/src/main/resources/templates/thymeleaf/application/application-overview.html#L197

Reply:
Man darf doch für sich selbst nichts machen oder?

Reply:
Ja, hätte gesagt das ist so für alle Rollen außer `OFFICE`. Weil initial gibt es ja nur den `OFFICE`-Benutzer.

Reply:
Office daef generell nicht genehmigen bzw ablehnen.

Reply:
Stimmt wohl. Wer genehmigt dann die Anträge in einer Firma mit genau einem `BOSS`?